playwright/tests/page-objects/canvas.ts

```diff
@@ -14,8 +14,8 @@ export class Canvas {
     return new Block(this.page, this.locator, { text });
   }
 
-  blockByNth(nth: number, layout: boolean = false) {
-    return new Block(this.page, this.locator, { nth, layout });
+  blockByNth(nth: number, layout: boolean = false, datatable: boolean = false) {
+    return new Block(this.page, this.locator, { nth, layout, datatable });
   }
```

Comment:
maybe it would make sense to introduce an optional object for this options, because then you don't need to add all parameters. 
```suggestion
blockByNth(nth: number, options?: { layout?: boolean, datatable?: boolean}) {
```